inlong-manager/manager-common/src/main/resources/iceberg-field-type-mapping.yaml

```diff
@@ -0,0 +1,66 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+source.type.to.target.type.converter:
+
+  - source.type: INT
+    target.type: INT
+
+  - source.type: INTEGER
+    target.type: INT
+
+  - source.type: FLOAT
+    target.type: FLOAT
+
+  - source.type: DOUBLE
+    target.type: DOUBLE
+
+  - source.type: LONG
+    target.type: LONG
+
+  - source.type: DECIMAL
+    target.type: DECIMAL
+
+  - source.type: BOOLEAN
+    target.type: BOOLEAN
+
+  - source.type: DATE
+    target.type: DATE
+
+  - source.type: TIME
+    target.type: TIME
+
+  - source.type: TIMESTAMP
+    target.type: LOCAL_ZONE_TIMESTAMP
+
+  - source.type: TIMESTAMPTZ
+    target.type: LOCAL_ZONE_TIMESTAMP
+
+  - source.type: TIMESTAMPLOCALTZ
+    target.type: LOCAL_ZONE_TIMESTAMP
+
+  - source.type: TIMESTAMP WITH LOCAL TIME ZONE
+    target.type: LOCAL_ZONE_TIMESTAMP
```

Comment:
source type is iceberg type. iceberg type have `timestamp with timezone`

https://iceberg.apache.org/docs/latest/spark-writes/

Reply:
just add TIMESTAMP WITH TIMEZONE、TIMESTAMP WITHOUT TIMEZONE field type.

Reply:
> just add TIMESTAMP WITH TIMEZONE、TIMESTAMP WITHOUT TIMEZONE field type.

TIMESTAMP WITHOUT TIMEZONE mapping to timestamp

Reply:
done